CMakeLists.txt

```diff
@@ -40,8 +40,5 @@ add_library(slowmokit
         src/slowmokit/methods/neighbors/bernoulli_nb.hpp
         src/slowmokit/methods/neighbors/bernoulli_nb/bernoulli_nb.hpp
         src/slowmokit/methods/neighbors/bernoulli_nb/bernoulli_nb.cpp
-        src/slowmokit/methods/metrics/classification_report.hpp
-        src/slowmokit/methods/metrics/classification_report.cpp
-        src/slowmokit/methods/neighbors/gaussian_nb.hpp
-        src/slowmokit/methods/neighbors/gaussian_nb/gaussian_nb.cpp
-        src/slowmokit/methods/neighbors/gaussian_nb/gaussian_nb.hpp)
```

Comment:
I guess these lines got deleted when you pulled, add these above your files

Reply:
> I guess these lines got deleted when you pulled, add these above your files

done